Description: Building expat against system's libxmltok library
 Provide support for building expat against the system's libxmltok library.
 .
 Not reported to upstream as this patch is mainly imported for being compliant
 with Debian policy (lintian error: embedded-library).
Forwarded: not-needed
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Last-Update: 2011-12-31
--- a/nx-X11/lib/expat/Imakefile
+++ b/nx-X11/lib/expat/Imakefile
@@ -19,7 +19,7 @@
 EXPATSRC = $(EXPATTOP)/lib
 EXPATINC = $(EXPATTOP)/lib
 
-INCLUDES = -I$(EXPATINC) -I$(EXPATTOP) -I.
+INCLUDES = -I/usr/include/xmltok -I/usr/local/include/xmltok -I$(EXPATINC) -I$(EXPATTOP) -I. 
 
 /*
  * Endianness (for short's)
@@ -31,8 +31,8 @@
 DEFINES = -DXML_BYTE_ORDER=0 -DX_BYTE_ORDER=$(X_BYTE_ORDER)
 REQUIREDLIBS=$(LDPRELIBS)
 
-SRCS = xmlparse.c xmltok.c xmlrole.c
-OBJS = xmlparse.o xmltok.o xmlrole.o
+#SRCS = xmlparse.c xmltok.c xmlrole.c
+#OBJS = xmlparse.o xmltok.o xmlrole.o
 
 HEADERS = expat.h
 
@@ -45,9 +45,9 @@
 
 DependTarget()
 
-LinkSourceFile(xmlparse.c,$(EXPATSRC))
-LinkSourceFile(xmltok.c,$(EXPATSRC))
-LinkSourceFile(xmlrole.c,$(EXPATSRC))
+#LinkSourceFile(xmlparse.c,$(EXPATSRC))
+#LinkSourceFile(xmltok.c,$(EXPATSRC))
+#LinkSourceFile(xmlrole.c,$(EXPATSRC))
 LinkSourceFile(expat.h,$(EXPATINC))
 
 
